src/main/java/com/microsoft/sqlserver/jdbc/SQLServerCallableStatement.java

```diff
@@ -1279,7 +1279,7 @@ private int findColumn(String columnName) throws SQLServerException {
                     // we should always have a procedure name part
                     metaQuery.append("@procedure_name=");
                     metaQuery.append(threePartName.getProcedurePart());
-                    metaQuery.append(" , @ODBCVer=3");
+                    metaQuery.append(" , @ODBCVer=3, @fUsePattern=0");
```

Comment:
Do you also want to consider applying the same change to SQLServerParameterMetadata's constructor? It also has sp_sproc_columns and won't have to use pattern matching.

Reply:
Done.